### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by WALL-E**
find quarrel in a straw

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,10 +69,17 @@ For user space, we need user application debuginfo like `redis-debuginfo-2.8.19-
 For *redhat*\* linux version, we can install as the following:
 
 ```bash
-yum install yum-utils #for debuginfo-install
-yum install systemtap
-yum install kernelname-devel-version
-debuginfo-install kernelname-version
+yum install -y redhat-lsb
```
Comment (detailyang) [inline README.md]: Can `lsb_release` be installed on all *redhat* forked system ? It is just a example to tell user get debuginfo -:)

File: README.md [hunk 2]
```diff
@@ -69,10 +69,17 @@ For user space, we need user application debuginfo like `redis-debuginfo-2.8.19-
...
+
+yum install -y kernel-devel-`uname -r`
+
+yum install -y http://debuginfo.centos.org/`lsb_release -sr|cut -d. -f1`/`uname -m`/kernel-debuginfo-common-`uname -m`-`uname -r`.rpm
+yum install -y http://debuginfo.centos.org/`lsb_release -sr|cut -d. -f1`/`uname -m`/kernel-debuginfo-`uname -r`.rpm
+
+yum install -y http://debuginfo.centos.org/`lsb_release -sr|cut -d. -f1`/`uname -m`/glibc-debuginfo-common-2.17-157.el7_3.1.x86_64.rpm
```
Comment (detailyang) [inline README.md]: the version of `glibc` should not be fixed.

File: README.md [hunk 3]
```diff
@@ -69,10 +69,17 @@ For user space, we need user application debuginfo like `redis-debuginfo-2.8.19-
...
+yum install -y http://debuginfo.centos.org/`lsb_release -sr|cut -d. -f1`/`uname -m`/glibc-debuginfo-2.17-157.el7_3.1.x86_64.rpm
```
Comment (detailyang) [inline README.md]: ditto

File: README.md [hunk 4]
```diff
@@ -69,10 +69,17 @@ For user space, we need user application debuginfo like `redis-debuginfo-2.8.19-
...
 ```bash
-yum install yum-utils #for debuginfo-install
-yum install systemtap
-yum install kernelname-devel-version
-debuginfo-install kernelname-version
+yum install -y redhat-lsb
+
+yum install -y kernel-devel-`uname -r`
+
+yum install -y http://debuginfo.centos.org/`lsb_release -sr|cut -d. -f1`/`uname -m`/kernel-debuginfo-common-`uname -m`-`uname -r`.rpm
```
Comment (detailyang) [inline README.md]: We can tell users to get debuginfo  package on [http://debuginfo.centos.org](http://debuginfo.centos.org) if they are in Centos

File: README.md (hunk 1 above)
Comment (WALL-E) [reply README.md]: I only tested on CentOS 7.2

this script is only support Redhat or CentOS, I will be to improve it

**Comment by WALL-E** [comment]
@detailyang  sorry, I update code.

**Comment by detailyang** [comment]
@WALL-E 
merged. many thanks!

**Pull request merged**